dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt

```diff
@@ -81,33 +78,34 @@ Melee thresholds are stronger than basics however are more taxing to use. These
     • *Examples:* <:assault:535532853979512842> <:destroy:535532879330148352> <:cane:535532878969438210> <:bloodtendrils:535532854327640064> <:slaughter:535532879237873666>
         - <:destroy:535532879330148352> *does stun however is more renowned for its damage.*
 
+⬥ __**Helper Thresholds:**__
+    • *Examples:* <:fbackhand:535532879346794516>
+
 ⬥ __**Utility Thresholds:**__
     • *Examples:* <:fbackhand:535532879346794516> <:gflurry:535532879283879977> <:quake:535532879506309150>
 
 ⬥ __**Subpar Thresholds:**__
     • *Examples:* <:stomp:535532879552315412>
 
-__**Auxiliary/Utility Examples:**__
+__**Helper/Utility Descriptions:**__
 ⬥ <:gflurry:535532879283879977> : Reduces remaining cooldown of <:Berserk:513190158468907012>
 ⬥ <:quake:535532879506309150> : Lowers the enemy's defenses for 60s.
-⬥ <:fbackhand:535532879346794516> : Stuns target for 3.6s.
+⬥ <:fbackhand:535532879346794516> : Stuns target for 3.6s; <:slice:535532879594258432> deals 40% more damage when stunned.
+
 
 > __**Ultimate Abilities**__
 .tag:ultimates
 .
-Melee Ultimates deal maximum damage and or elevate other abilities. They typically have 1-3 purposes:
+Melee ultimates deal maximum damage and or elevate other abilities. These abilities have 1-3 purposes:
 ⬥ Sacrifice 100% adrenaline
 ⬥ Deal devastating damage
-⬥ Provide a longstanding buff
+⬥ Provide an extended buff
 
 ⬥ __**Power Ultimates:**__
     • *Examples:* <:overpowerigneous:959089455640215633>
 
-⬥ __**Auxiliary Ultimates:**__
-    • *Examples:* <:Berserk:513190158468907012> <:meteorstrike:535532879359377439>
-
 ⬥ __**Utility Ultimates:**__
-    • *Examples:* <:pulverise:535532879053062146>
+    • *Examples:* <:pulverise:535532879053062146> <:Berserk:513190158468907012> <:meteorstrike:535532879359377439>
```

Comment:
Here maybe give a breakdown of the utility components like you did for thresholds? So maybe something like

__**Utility Examples:**__
⬥ <:slaughter:535532879237873666> + <:kick:535532879132885025> → <:slaughter:535532879237873666> damage is tripled when target moves.
⬥ <:surge:535533810004262912> + <:bd:535532854281764884> <:dive:1049378668197195808> → <:surge:535533810004262912> length is extended; Can redirect destination.